ui/src/app/cohort-review/set-annotation-item/set-annotation-item.component.ts

```diff
@@ -43,9 +43,13 @@ export class SetAnnotationItemComponent {
   edit(): void {
     this.editing = true;
     this.name.setValue(this.definition.columnName);
+    this.setFocus();
+  }
+
+  setFocus() {
```

Comment:
Can we/should we put setFocus in test-helpers? Do we think it is going to see a lot of use in many different components?

Reply:
It probably won't get a lot of use actually, its a pretty specific condition that this solves.  We'd have to do some work to remove it from the component b/c it relies on closures, but if we find that its being used a lot we could do that.

Why would it go into test-helpers though?  Its a component method.